sdk/src/resource/resource.cc

```diff
@@ -0,0 +1,69 @@
+#include "opentelemetry/sdk/resource/resource.h"
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/sdk/resource/resource_detector.h"
+#include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace resource
+{
+
+const std::string TELEMETRY_SDK_LANGUAGE = "telemetry.sdk.language";
```

Comment:
Name the constants like `kTelemetrySdkLanguage`?

Reply:
Thanks. fixed now :)